tests/pyrog/factories.py

```diff
@@ -83,7 +83,6 @@ class Params:
         with_join = factory.Trait(join=factory.SubFactory("tests.pyrog.factories.JoinFactory"))
 
     id = factory.Sequence(lambda n: f"column_id_{n:04d}")
-    input = factory.SubFactory(InputFactory)
```

Comment:
You removed the input on the column but what about adding the column to the input factory ?

Reply:
Inputs don't necessary have columns so I decided not to add any. We could still parametrize the factory but as this model will be removed in a few PRs...